manifests/apt.pp

```diff
@@ -13,31 +13,19 @@
     }
   }
 
-  package { $::duo_unix::duo_package:
-    ensure  => $package_state,
-    require => [
-      File[$repo_file],
-      Exec['Duo Security GPG Import'],
-      Exec['duo-security-apt-update']
-    ]
-  }
-
-  file { $repo_file:
-    owner   => 'root',
-    group   => 'root',
-    mode    => '0644',
-    content => "deb ${repo_uri}/${facts['operatingsystem']} ${facts['lsbdistcodename']} main",
-    notify  => Exec['duo-security-apt-update']
-  }
-
-  exec { 'duo-security-apt-update':
-    command     => '/usr/bin/apt-get update',
-    refreshonly => true
-  }
+  if $::duo_unix::manage_repo {
```

Comment:
I don't disagree with this line, but I also can't find any uses of 'manage_repo' that are about managing apt sources.  So I think there's a slight bit of surprise to this behavior.  This, IMO, warrants a comment against the manage_repo param.

Reply:
SGTM, replaced all occurrences `${module_name}` with `duo`.